doc/configure

```diff
@@ -46,6 +46,16 @@ RULES = [{
     "name": "sc_to_roff",
     "description": "converting ${man-name} to roff",
     "command": "scdoc < ${in} > ${out}"
+}, {
+    "name": "voluptuous_to_roff",
```

Comment:
Seeing few occurrences of `roff`. Should it be `groff` instead? 
`sed s/roff/groff/g doc/configure`